### PR TITLE
Use styled-system variant to create Text component

### PR DESCRIPTION
**Pull request by okonet**
subj

### CODE REVIEW

File: src/exercises/2-Design_Tokens/theme.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const theme = {
 	colors: {
-		myfavoritecolor: '#f39037',
+		myfavoritecolor: '#400',
```
Comment (sapegin) [inline src/exercises/2-Design_Tokens/theme.js]: This looks unrelated ;-)

Comment (okonet) [reply src/exercises/2-Design_Tokens/theme.js]: Reverted the change.
